source-academy/cadet-frontend
# Update deprecated ag-grid props

## UI/UX: Update deprecated ag-grid props
Summary: Code quality improvements.

### Description
Some of the existing grid props and column definition props of the ag-grid `AgGridReact` component were deprecated in v19 and v20, as shown by these warning messages in the console.

![image](https://user-images.githubusercontent.com/44989315/84522123-a5ba1200-ad08-11ea-9e1e-788f4363ba42.png)

### Changelog
- Replace deprecated ag-grid props with their current recommended equivalents

### Verification instructions
1. Navigate to any page containing an ag-grid datagrid, e.g. the Grading page - it should load without any warning messages about deprecated props

Last updated 13 Jun 2020, 12:00AM
